src/components/IndividualPageContent/ContentRow.tsx

```diff
@@ -9,22 +9,27 @@ type ContentRowProps = {
   value: React.ReactNode;
   tooltip?: React.ReactNode;
   i?: any;
+  container?: boolean;
+  sx?: SxProps<Theme>;
 };
 
 export default function ContentRow({
   title,
   value,
   tooltip,
   i,
+  container,
+  sx,
 }: ContentRowProps) {
   return (
     <Box>
       <Grid
-        container
+        container={container && true}
```

Comment:
why is the `&& true`?

Reply:
cuz not all places will pass in `container` variable so that if its null, container will equal to true, which's the need for other use cases.